webapp/src/components/sidebar_buttons/sidebar_buttons.jsx

```diff
@@ -226,7 +226,7 @@ const getStyle = makeStyleFromTheme((theme) => {
             marginBottom: '5px',
             display: 'flex',
             alignItems: 'center',
-            justifyContent: 'space-around',
+            justifyContent: 'space-between',
```

Comment:
That doesn't seem to resolve the issue when there are multiple teams. In that case `containerTeam` is used instead of `containerHeader`. How should the CSS look in that case?